test.py

```diff
@@ -1,4 +1,10 @@
+import asyncio
+
 import plotly.express as px
+import kaleido
 
 fig = px.scatter(px.data.iris(), x="sepal_length", y="sepal_width", color="species")
-fig.write_image("/output/figure.png", engine="kaleido")
```

Comment:
This API will be coming back in a few weeks, but the lower level API is here and it is stable.